jobs/rootfs-builder.jpl

```diff
@@ -123,10 +123,34 @@ node("debos && docker") {
             }
         }
 
+        stage("scp") {
+            sshagent(credentials : ['jenkins-chromeos-storage-private-key']) {
+                sh(script: """\
+ssh \
+-o StrictHostKeyChecking=no \
+-p 22022 \
+chromeos@storage.chromeos.kernelci.org 'mkdir ~/images/rootfs/chromeos/${pipeline_version} || true'
+"""
+                  )
+                sh(script: """\
+scp \
+-o StrictHostKeyChecking=no \
+-P 22022 \
+-r \
+kernelci-core/${pipeline_version}/_install_/* \
+chromeos@storage.chromeos.kernelci.org:\
+~/images/rootfs/chromeos/${pipeline_version}
+"""
+                  )
+            }
+        }
+/* Upload stage not working, as kci_rootfs http(s) upload fails for files >1GB */
+/*
         stage("Upload") {
             timeout(time: 30, unit: 'MINUTES') {
                 upload(config, pipeline_version, kci_core, rootfs_type, arch);
             }
         }
+*/
```

Comment:
I think we can just delete this now rather than keep unused code in the file.

Reply:
Done